packages/generator/src/entity/class.ts

```diff
@@ -27,6 +27,18 @@ export function entityClass(
   return {
     kind: StructureKind.Class,
     name: `${entity.className}<T extends DeSerializers = DefaultDeSerializers>`,
+    ctors: [
+      {
+        parameters: [
+          {
+            name: '_entityApi',
```

Comment:
[q] why `_` prefix for the name? Is this our convention for private things?

Reply:
This seems to be the convention for properties on entities to avoid name clashes.